src/App/index.js

```diff
@@ -1,13 +1,23 @@
 
-import React, {Component} from "react";
-import style from "./style.scss";
+import React, {Component, PropTypes} from 'react';
+import {Provider} from 'react-redux';
+import style from './style.scss';
+
+import store from './store';
 
 export default class App extends Component {
+    static propTypes = {
```

Comment:
I don't think the props `children` needed to be declared here. prefer to remove it.